src/main/java/fi/okm/jod/yksilo/entity/Ammatti.java

```diff
@@ -45,6 +38,9 @@ public class Ammatti {
   @BatchSize(size = 1000)
   private Map<Kieli, Kaannos> kaannos;
 
+  @ManyToMany(mappedBy = "ammattiKiinnostukset")
+  private Set<Yksilo> kiinnostuneet;
+
```

Comment:
This association must be unidirectional (Yksilo->Ammatti), please remove this. Note that the cardinality Ammatti->Yksilo is essentially unbounded (in theory all users could be interested in the same ammatti).